src/planner.cc

```diff
@@ -663,175 +668,205 @@ void Planner::Impl::bring_path_to_halt() {
   issue_motor_move_if_possible(true);
 }
 
+// Compute the backard pass of <segment_to_plan> given the
```

Comment:
The word _Backward_ and _Forward_ might make sense in the way we compute them when looking through our queue, but that is also just an artifact how we organized our queue.
Maybe we should name these functions in the functionality they do, e.g. function names such as `DecelerationPlanSegment()` and `AccelerationPlanSegments()` ? Then when reading the loop itself, it then also emphasises to the reader which directions our queue is set up.
Then there is less ambiguity as looking at the names of the functions explains

Reply:
(solved)